### ISSUE TITLE
Don't show camel case flags in autocompletion

### ISSUE THREAD

**Issue by jvican** [scalacenter/bloop#412]
Autocompletions get the flag names from caseapp directly without converting camel case to the actual CLI format. An example is `configDir`, which should be shown as `--config-dir`:
  
```
jvican in /data/rw/code/scala/loop                                                                                                                                                                [13:57:38] 
> $ bloop compile backend -                                                                                                                                                        [±topic/fix-export-sbt ●]
--configDir    -- File path to the bloop config directory.
--incremental  -- Compile the project incrementally. By default, true.
--project      -- The project to compile (will be inferred from remaining cli args).
--reporter     -- Pick reporter to show compilation messages. By default, bloop's used.
--verbose      -- If set, print out debugging information to stderr.
--version      -- If set, print the about section at the beginning of the execution.
--watch        -- Run the command when projects' source files change. By default, false.
--threads
```

**Comment by rberenguel** [comment]
Hi @jvican have never touched caseapp, but I'd guess it has a way of doing this as an interface... otherwise it sounds like downcasing and adding a dash where needed is what is required. I'll give this a look if no-one else picks it up before I can, I want to confirm I have a local setup allowing me to build and test bloop so I can help with something slightly harder (installed bloop today and I'm pretty sure I'll be using it a lot more...)

**Comment by Duhemm** [comment]
Thanks for volunteering @rberenguel ! We have some developer documentation here: https://scalacenter.github.io/bloop/docs/developer-documentation/.

The code that you'd need to change to fix this issue is probably here: https://github.com/scalacenter/bloop/blob/09d685fe1ee79a106ef44d3a2107f711ccfa5abe/frontend/src/main/scala/bloop/engine/Interpreter.scala#L224-L230

**Comment by olafurpg** [comment]
There are tons of ways to convert camel to kebab, but I'll throw in my implementation from here just in case https://github.com/olafurpg/metaconfig/blob/490e4cee64d9f4a265ce789102cc2be5fa0609fe/metaconfig-core/shared/src/main/scala/metaconfig/internal/Case.scala#L19

**Comment by rberenguel** [comment]
Hi @Duhemm thanks for the pointers, but it's pretty well addressed in the readme and docs, good work there for making it easy for contributors. 

Thanks @olafurpg I'll have a look!

**Comment by jvican** [comment]
Fixed by #423 :tada:
